lib/services/queue/bull/index.js

```diff
@@ -110,10 +110,12 @@ export const subscribe = ({
   });
 };
 
-export const unsubscribeAll = () => {
+export const unsubscribeAll = async () => {
   const queues = map(Bull.queues, queue => queue);
-  forEach(queues, ((queue) => {
-    queue.close(true);
-  }));
+
+  for (const queue of queues) {
```

Comment:
Avoid use of for loop. Use bluebird: http://bluebirdjs.com/docs/api-reference.html. Promise.map maybe.